new-dti-website/components/slideshow.tsx

```diff
@@ -19,10 +19,10 @@ const imageNames = [
 const ImageHeader: React.FC<{ imageName: string }> = ({ imageName }) => (
   <div className="md:p-4 xs:p-2 flex items-center rounded-[20px] gap-2">
     <img src="/images/folder_icon.png" alt="Folder" className="xs:h-3 md:h-6" />
-    <span className={`font-medium xs:text-[10px] md:text-[22px] ${ibm_plex_mono.className}`}>
+    <span className={`font-medium xs:text-[8px] md:text-[16px] ${ibm_plex_mono.className}`}>
       cornell-dti
     </span>
-    <span className={`ml-auto font-medium md:text-[14px] xs:text-[7px] ${ibm_plex_mono.className}`}>
+    <span className={`ml-auto font-medium md:text-[16px] xs:text-[8px] ${ibm_plex_mono.className}`}>
```

Comment:
`8px` is too small and isn't a safe/accessible font size, especially for mobile.

I'd say keep it `16px` for all screen sizes.


I tested it at `16px` on a mobile device (@ `375px` screen width) and even when making the icon the same size, it should work for the slide with the longest text ("collaboration.png")

![image](https://github.com/user-attachments/assets/7b4ce71f-c30f-4c0e-8a4e-bfacc755f8b8)